packages/foam-core/src/markdown-provider.ts

```diff
@@ -44,7 +48,7 @@ export function createNoteFromMarkdown(
   let start: Point = { line: 1, column: 1, offset: 0 }; // start position of the note
   visit(tree, node => {
     if (node.type === 'yaml') {
-      frontmatter = parseYAML(node.value as string);
+      frontmatter = parseYAML(node.value as string) ?? {}; // parseYAML returns null if the frontmatter is emtpy
```

Comment:
cc @anku255

Reply:
fixed!

Reply:
Fun story:
I kept looking at the comment, and the code, and the comment.. and not understand what it was about.
Then I looked at it again, comment and code, and still nothing.
So I thought "let's see if Ankit understands this" because I clearly don't! (so I cc'ed him)
After he implemented the fix, I kept looking at the code, the comment, the code - it took me 3 more times to finally understand there was a typo 🤣

Reply:
🤣🤣🤣🤣